server/container_remove.go

```diff
@@ -23,7 +23,7 @@ func (s *Server) RemoveContainer(ctx context.Context, req *pb.RemoveContainerReq
 	}
 
 	cState := s.runtime.ContainerStatus(c)
-	if cState.Status == oci.ContainerStateCreated || cState.Status == oci.ContainerStateRunning {
+	if cState.Status == oci.ContainerStateRunning {
```

Comment:
Created still means that the process is started (but not exec'ed) in OCI. We could clarify that in the spec perhaps.